keyboards/mechanicalkeyboards/point65/info.json

```
@@ -0,0 +1,72 @@
+{
+    "manufacturer": "Mechanical Keyboards",
+    "maintainer": "C44Supra",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
```

moved from keyboard.json
```suggestion
    "diode_direction": "COL2ROW",
    "matrix_pins": {
        "cols": ["GP28", "GP27", "GP26", "GP29", "GP0", "GP11", "GP10", "GP9", "GP8", "GP7", "GP6", "GP5", "GP4", "GP3", "GP2", "GP1"]
    },
```